server.js

```diff
@@ -53,23 +50,26 @@ function LimitdServer (options) {
   this._server = net.createServer(this._handler.bind(this));
   enableDestroy(this._server);
 
-  this._server.on('error', function (err) {
-    self.emit('error', err);
+  this._server.on('error', (err) => {
+    this.emit('error', err);
   });
 
-  var dbConfig = { types: this._config.buckets };
+  const dbConfig = { types: this._config.buckets };
 
   if (typeof this._config.db === 'string') {
     dbConfig.path = this._config.db;
-  } else if(typeof this._config.db === 'object') {
+  } else if (typeof this._config.db === 'object') {
     Object.assign(dbConfig, this._config.db);
   }
 
   this._db = new LimitDB(dbConfig);
-
   this._db
-    .on('ready', () => logger.info({ path: dbConfig.path }, 'Database ready.'))
-    .on('error', err => this.emit('error', err))
+    .on('ready', () => {
+      logger.info({ path: dbConfig.path }, 'Database ready.');
+    })
+    .on('error', (err) => {
+      logger.error({ err });
```

Comment:
add a message to the error or a log type (or both) so we can identify where it comes from